pkg/features/kube_features_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"testing"
+)
+
+func TestKubeFeatures(t *testing.T) {
+	features := utilfeature.DefaultMutableFeatureGate.GetAll()
```

Comment:
I think you can call `utilfeature.DefaultFeatureGate.KnownFeatures()` instead and not need the verify test change

Reply:
This method skips some gates(the ones that have `PreRelease` - `GA` or `Deprecated`). Should we ignore them?

Reply:
hmm, no we want those as well.

in that case, do `utilfeature.DefaultFeatureGate.DeepCopy().GetAll()` to avoid needing the verify test change

Reply:
Ok, I've implemented it with `utilfeature.DefaultFeatureGate.DeepCopy().GetAll()`